### ISSUE TITLE
Multiple Layer Representations using Tensorflow Hub

### ISSUE THREAD

**Issue by rajicon** [google-research/bert#768]
Is there any simple way to get each layer representation of each word using Tensorflow Hub?  Looking here https://tfhub.dev/google/bert_uncased_L-12_H-768_A-12/1 it seems that "sequence_output" will get one representation per token (I assume this is the last layer).

**Comment by Cumberbatch08** [comment]
> Is there any simple way to get each layer representation of each word using Tensorflow Hub? Looking here https://tfhub.dev/google/bert_uncased_L-12_H-768_A-12/1 it seems that "sequence_output" will get one representation per token (I assume this is the last layer).

yes, I can get the word embedding in tensorflow hub bert by 'sequence_output'. but how to get all layers word vector? 
In other words, how to get the word embeddings from layer 1 to the layer 12 ?
